spec/requests/admin/staff_action_logs_controller_spec.rb

```diff
@@ -31,6 +31,26 @@
       )
     end
 
+    it 'generates logs with pages' do
+      1.upto(300).each do |idx|
```

Comment:
This is a lot of records to create. Is there any way to test it with fewer? I'm just worried about a slow test here.